### PR TITLE
adds preferred email to user admin form.

### LINKED ISSUE #4376: Provide for a "Preferred Email" field
Add a 2ndary email field for preferred email. This will be editable via the MyProfile page by the user, as well as from the admin profile page. When the preferred email is populated, it will be used for all communication from the system, and will not be updated by the EDS sync

### PR DESCRIPTION
**Pull request by stopfstedt**
fixes #4376

### CODE REVIEW

File: app/components/user-profile-bio.js [hunk 1]
```diff
@@ -204,6 +226,7 @@ export default Component.extend(ValidationErrorDisplay, Validations, {
       const firstName = this.firstName;
       const lastName = this.lastName;
       const email = this.email;
+      const preferredEmail = this.preferredEmail;
```
Comment (jrjohnson) [inline app/components/user-profile-bio.js]: I don't think this property will every by part of the sync. I'd have to look further, but I don't think we return a full user record from the sync run, it's an object based on what is in LDAP.

Comment (stopfstedt) [reply app/components/user-profile-bio.js]: thanks! taking it out.

**Comment by stopfstedt** [comment]
> Can you add a test for removing / empty preferred email? I think the validation for email accounts for blank, but I don't _know_ that it does.

good catch. fixed and backed up with a test.
